Phase4/Project/Manager.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Project
+{
+    public class Manager
+    {
+        private const string StudentsJsonPath = "Students.json";
+        private const string ScoresJsonPath = "Scores.json";
+        private const int TopListSize = 3;
+        public void run()
+        {
+            var students = JsonFileReader.Read<Student[]>(StudentsJsonPath);
+            var scores = JsonFileReader.Read<StudentScore[]>(ScoresJsonPath);
+            Ranker ranker = new Ranker(students, scores);
+
+            var topStudents = ranker.GetNTopStudents(TopListSize);
+            var topStudentsScore = ranker.GetNTopStudentsScores(TopListSize);
+            for (int i = 0; i < topStudents.Count; i++)
+            {
+                var student = topStudents[i];
+                var score = topStudentsScore[i];
+                Console.WriteLine(student.FirstName + " " + student.LastName + " " + score);
+            }
+        }
```

Comment:
One of the most powerful methods of Linq is Join(), which can be used to combine two lists and create a new one with specified behavior.
I just recommend that you take a look at this method. :-)